### ISSUE TITLE
Action Bar, Action Group and Action - Provide valid, concise, and meaningful alternative text for image buttons - (2036569763)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-design-system#7777]
### Violation: 
 Provide valid, concise, and meaningful alternative text for image buttons 

![image](https://github.com/Esri/calcite-design-system/assets/5023024/67704f63-8f56-4029-8b7e-bccd7b4db218)

### WCAG Reference: 
- [1.1.1 Non-text Content](https://www.w3.org/WAI/WCAG21/Understanding/non-text-content.html)
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Severity: 
 6 

### Media Type: 
 Forms 

--- 
### Areas for Remediation: 
- Title: Action Bar, Action Group and Action - Provide valid, concise, and meaningful alternative text for image buttons - (2036569763) 
- Module: Action Bar, Action Group and Action 
**Issue**
There are expand/collapse image buttons that do not provide the complete accessible names. The expand/collapse state is defined using an accessible name. Examples include:

- Action bar
- Action bar small scale
- Action bar expanded
- Action bar with groups

**User Impact**
Without meaningful alternative text, any information conveyed by these image buttons will not be conveyed to screen reader users and they will be announced as unlabeled buttons.

**Code Reference** 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Expand" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  </div>
  <div class="text-container">Expand</div>
</button>
 ``` 
**Suggestion**
Ensure that image buttons provide meaningful alternative text. This can be achieved by providing an aria-label for ``` <button> ``` elements.

**Compliant Code Example** 
 ```
<button aria-busy="false" aria-disabled="false" aria-label="Action bar Expand" class="button">
  <div aria-hidden="true" class="icon-container">
    (...)
  </div>
  <div class="text-container">Expand</div>
</button>
 ```
 
--- 
### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by driskull** [comment]
I think the label property solves this. could be an app issue.

**Comment by geospatialem** [comment]
More context is now provided as of `1.11.0` and the icon is now displayed with an `aria-hidden` attribute so AT skips over it.
